jerry-core/ecma/builtin-objects/ecma-builtin-array-prototype.c

```diff
@@ -1890,19 +1890,49 @@ ecma_builtin_array_prototype_object_last_index_of (ecma_value_t this_arg, /**< t
   return ret_value;
 } /* ecma_builtin_array_prototype_object_last_index_of */
 
+
 /**
- * The Array.prototype object's 'every' routine
- *
- * See also:
- *          ECMA-262 v5, 15.4.4.16
- *
- * @return ecma value
- *         Returned value must be freed with ecma_free_value.
+ * If the argument is convertible to boolean true, returns ECMA_VALUE_TRUE,
+ * otherwise returns ECMA_VALUE_EMPTY.
```

Comment:
To specify the return value we usually use the `@returns` element in the docs. (please see a method above)